wgpu-types/src/features.rs

```diff
@@ -1301,6 +1304,19 @@ bitflags_array! {
         /// This is a web and native feature.
         const TEXTURE_COMPRESSION_ASTC = WEBGPU_FEATURE_TEXTURE_COMPRESSION_ASTC;
 
+
+        /// Allows the 3d dimension for textures with ASTC compressed formats.
+        ///
```

Comment:
would be nice to mention on the above feature that volume textures aren't included unless `TEXTURE_COMPRESSION_ASTC_SLICED_3D` is enabled.

Wondering what to do with the non-webgpu `_HDR` variant, seems a bit underspeced 🤔. I'm fine with ignoring that for now.

Reply:
@Wumpf makes sense, does the new commit look sufficient?